### PR TITLE
DEP Remove modules that have no tests

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/503

- Adds testsuites node for phpunit 9
- Remove vendor/symbiote/silverstripe-errorpage/tests
- Remove vendor/colymba/gridfield-bulk-editing-tools/tests
- Remove vendor/dnadesign/silverstripe-elemental-subsites/tests
- Remove vendor/bringyourownideas/silverstripe-composer-security-checker/tests

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,144 +1,140 @@
...
+            <directory>vendor/silverstripe/mimevalidator/tests</directory>
+        </testsuite>
 
-    <testsuite name="cwp-recipe-cms">
-        <directory>vendor/cwp/cwp-core/tests</directory>
-        <directory>vendor/cwp/cwp/tests</directory>
-        <directory>vendor/cwp/cwp-pdfexport/tests</directory>
-        <directory>vendor/silverstripe/html5/tests</directory>
-        <directory>vendor/symbiote/silverstripe-gridfieldextensions/tests</directory>
-        <directory>vendor/symbiote/silverstripe-errorpage/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist:65]: This was probably a mistake and was probably meant to test `silverstripe/errorpage` which does have tests. Can we swap it in instead?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -1,144 +1,140 @@
...
+            <directory>vendor/silverstripe/graphql/tests/</directory>
+            <directory>vendor/silverstripe/siteconfig/tests/php/</directory>
+            <directory>vendor/silverstripe/reports/tests/</directory>
+            <!-- CMS needs to run last because the SearchFormTest enables FulltextSearchable  -->
+            <directory>vendor/silverstripe/cms/tests/</directory>
+        </testsuite>
 
-    <!-- Optional CWP modules that aren't in recipes -->
-    <testsuite name="Default">
-        <directory>mysite/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist:38]: This is removed because we don't have any test there.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -1,144 +1,140 @@
...
-        <directory>vendor/cwp/agency-extensions/tests</directory>
-        <directory>vendor/silverstripe/subsites/tests</directory>
-        <directory>vendor/silverstripe/ckan-registry/tests</directory>
-        <directory>vendor/silverstripe/controllerpolicy/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist:42]: Controller policy is being removed from the sink.

File: phpunit.xml.dist [hunk 4]
```diff
@@ -1,144 +1,140 @@
...
-    <testsuite name="recipe-blog">
-        <directory>vendor/silverstripe/blog/tests</directory>
-        <directory>vendor/silverstripe/lumberjack/tests</directory>
-        <directory>vendor/silverstripe/widgets/tests</directory>
-        <directory>vendor/silverstripe/content-widget/tests</directory>
-        <directory>vendor/silverstripe/spamprotection/tests</directory>
-        <directory>vendor/silverstripe/akismet/tests</directory>
-        <directory>vendor/silverstripe/comments/tests</directory>
-        <directory>vendor/silverstripe/comment-notifications/tests</directory>
-        <directory>vendor/colymba/gridfield-bulk-editing-tools/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist:93]: Doesn't have any CI

File: phpunit.xml.dist [hunk 5]
```diff
@@ -1,144 +1,140 @@
...
-    </testsuite>
+        <testsuite name="recipe-collaboration">
+            <directory>vendor/silverstripe/contentreview/tests</directory>
+            <directory>vendor/silverstripe/sharedraftcontent/tests</directory>
+            <directory>vendor/symbiote/silverstripe-advancedworkflow/tests</directory>
+        </testsuite>
 
-    <testsuite name="recipe-content-blocks">
-        <directory>vendor/dnadesign/silverstripe-elemental/tests</directory>
-        <directory>vendor/dnadesign/silverstripe-elemental-subsites/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist:104]: This module is not part of the sink.

File: phpunit.xml.dist [hunk 6]
```diff
@@ -1,144 +1,140 @@
...
+        </testsuite>
 
-    <testsuite name="recipe-reporting-tools">
-        <directory>vendor/silverstripe/externallinks/tests</directory>
-        <directory>vendor/silverstripe/reports/tests</directory>
-        <directory>vendor/silverstripe/securityreport/tests</directory>
-        <directory>vendor/silverstripe/sitewidecontent-report/tests</directory>
-        <directory>vendor/bringyourownideas/silverstripe-maintenance/tests</directory>
-        <directory>vendor/bringyourownideas/silverstripe-composer-update-checker/tests</directory>
-        <directory>vendor/bringyourownideas/silverstripe-composer-security-checker/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist:124]: composer-security-checker was removed from the sink a while back

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Swap `symbiote/silverstripe-errorpage` with `silverstripe/errorpage`. Everything else looks fine.

Left notes on the removed module for posterity in case someone ever wonders why those got removed.

File: phpunit.xml.dist (hunk 1 above)
Comment (emteknetnz) [reply phpunit.xml.dist:65]: Done

**Review by maxime-rainville — APPROVED** [review]
Looks good ... I'll merge now since this change won't break or fix build by itself.

**Pull request merged**